themes/wilson_theme_starterkit/wilson_theme_starterkit.libraries.yml

```diff
@@ -22,6 +22,16 @@ adaptive-panels:
     - core/drupal
     - wilson_theme_starterkit/global
 
+aos:
+  css:
+    theme:
+      https://unpkg.com/aos@2.3.1/dist/aos.css: { type: external, minified: true }
+  js:
+    https://unpkg.com/aos@2.3.1/dist/aos.js: { type: external, minified: true }
```

Comment:
Having given this some thought and consulted Jake, I've come up with a small bit of config in package.json which allows us to declare specific files from `node_modules` which should be copied to `dist` at build time. These can then be used as normal in the `*.libraries.yml` declarations. Included some content about this in the Readme and switched AOS and GlideJs over to npm packages, rather than CDN includes.